README.md

```diff
@@ -153,6 +153,12 @@ These benchmarks allow you to benchmark any Ingest Node Pipelines defined by you
 
 For details on how to configure pipeline benchmarks for a package, review the [HOWTO guide](./docs/howto/pipeline_benchmarking.md).
 
+#### Rally Benchmarks
+
+These benchmarks allow you to benchmark an integration corpus with rally.
+
+For details on how to configure rally benchmarks for a package, review the [HOWTO guide](./docs/howto/rally_benchmarking.md).
```

Comment:
Is this link correct? Trying to find it in this PR.

Reply:
the link is correct. but I haven't yet written the docs :)

Reply:
Can we add these to the PR? I was looking for these as I hit some issues testing the PR. Will comment on more on the issue I hit soon.

Reply:
yes, it's planned to add to this PR: in the description I listed `missing docs` :)

Reply:
Will give it a try when there are docs :slightly_smiling_face: